pyvcloud/vcd/system.py

```diff
@@ -57,13 +59,64 @@ def delete_org(self, org_name, force=None, recursive=None):
         """
         Delete an organization.
         :param org_name: (str): name of the org to be deleted.
-        :param force: (bool): pass force=True  along with recursive=True to 
-        remove an organization and any objects it contains, regardless of 
+        :param force: (bool): pass force=True  along with recursive=True to
+        remove an organization and any objects it contains, regardless of
         their state.
-        :param recursive: (bool): pass recursive=True  to remove an 
-        organization and any objects it contains that are in a state that 
+        :param recursive: (bool): pass recursive=True  to remove an
+        organization and any objects it contains that are in a state that
         normally allows removal.
         """  # NOQA
         org = self.client.get_org_by_name(org_name)
         org_href = get_admin_href(org.get('href'))
         return self.client.delete_resource(org_href, force, recursive)
+
+    def list_provider_vdcs(self):
```

Comment:
Can we add docstring to the newly added methods to make it consistent with other methods in the class?

Reply:
done